### DISCUSSION TITLE
ODAFC

### DISCUSSION THREAD

**Discussion by sree9578** [mozman/ezdxf#628] [Q&A] [answered]
Hi team,

  i am having a doubt whether the ODAFC (ezdxf add on ) is a part of MIT license or i need to get membership from ODA for using it commercial?

thanks..it would be great to get a faster response.. @mozman

**Comment by mozman** [comment, marked as answer]
The ODA file converter is not part of `ezdxf`, you have to install it by yourself from the ODAFC [website](https://www.opendesign.com/guestfiles/oda_file_converter). (fast enough?)

**Reply by sree9578** [reply]
Yup...thanks for the help
